### PR TITLE
Flatpak improvements

### PR DESCRIPTION
**Pull request by bilelmoussaoui**
Helps somehow with #2; once the manifest is converted to json things should work fine everywhere :) 

The Github Action will only run on your repository once merged, but you can see the results in my local branch here 
https://github.com/bilelmoussaoui/Plots/runs/1676607500

please note that the latest commit has to be reverted before merging and it was added just to run the CI on my fork

### CODE REVIEW

**Comment by bilelmoussaoui** [comment]
Should be ready to go from my side, it works just fine with GNOME Builder & Visual Studio Code (with the Flatpak extension)

File: com.github.alexhuntley.Plots.json [hunk 1]
```diff
@@ -0,0 +1,50 @@
+{
+  "app-id": "com.github.alexhuntley.Plots",
+  "runtime": "org.gnome.Platform",
+  "runtime-version": "3.38",
+  "sdk": "org.gnome.Sdk",
+  "command": "plots",
+  "finish-args": [
+    "--socket=fallback-x11",
```
Comment (alexhuntley) [inline com.github.alexhuntley.Plots.json:8]: For some reason, the app won't start on x11 for me when using fallback-x11 
```suggestion
    "--socket=x11",
```

**Review by alexhuntley — CHANGES_REQUESTED** [review]
Wow, this looks good! Only problem is --socket=fallback-x11 doesn't work on my machine...

File: com.github.alexhuntley.Plots.json (hunk 1 above)
Comment (bilelmoussaoui) [reply com.github.alexhuntley.Plots.json:8]: there's probably an issue with your machine then :P do you have any output when you run it from the terminal using `flatpak run com.github.alexhuntley.Plots`?

Comment (alexhuntley) [reply com.github.alexhuntley.Plots.json:8]: Just tried it again and it worked fine, not sure why. The output before was something about not being able to connect to the display.

Comment (bilelmoussaoui) [reply com.github.alexhuntley.Plots.json:8]: Ah, classic one. The issue comes from something else but I couldn't find the time to track what causes it exactly and I'm not able to reproduce it for a long time now. I think it's might be caused by gdm or whatever that handles the login to specific session on your system

**Pull request merged**